src/main/kotlin/com/mineinabyss/launchy/logic/LaunchyState.kt

```diff
@@ -163,7 +161,7 @@ class LaunchyState(
             downloadURLs[mod] = mod.url
             save()
 
-            if (mod.configUrl != null && enabledConfigs.contains(mod)) {
+            if (mod.configUrl.isNotBlank() && (mod in enabledConfigs)) {
```

Comment:
We specifically use null so you get compiler safety that ensures you checked the thing isn't null before trying to use it. `isNotBlank` doesn't do that for us, revert

Reply:
well by default it is "" which isnt null it is blank
so only case it can be null is if u change the default to be nullable
rn it can never be null only blank aka "" or " " etc

Reply:
@0ffz